packages/core/src/components/tag/tag.tsx

```diff
@@ -50,6 +50,13 @@ export class Tag extends React.Component<ITagProps, {}> {
             </span>
         );
     }
+
+    private onRemoveClick = (e: React.MouseEvent<HTMLButtonElement>) => {
+        const { onRemove } = this.props;
+        if (onRemove) {
+            onRemove(e, this.props);
```

Comment:
@cmslewis okay, do you all prefer this style? I slightly prefer the existing (clear what is happening without prior knowledge of what `safeInvoke` does, very slightly more performant since it doesn't add another call), but fine with changing it